pennylane/ops/qubit/special_unitary.py

```diff
@@ -224,6 +224,11 @@ class SpecialUnitary(Operation):
     >>> qml.ops.qubit.special_unitary.pauli_basis_strings(2) # 4**2-1 = 15 Pauli words
     ['IX', 'IY', 'IZ', 'XI', 'XX', 'XY', 'XZ', 'YI', 'YX', 'YY', 'YZ', 'ZI', 'ZX', 'ZY', 'ZZ']
 
+    .. seealso::
+
+            For more details on using this operator in applications, see the
+            :doc:`in-depth demonstration <demos/tutorial_here_comes_the_sun>`.
```

Comment:
Hey @isaacdevlugt, would you like to add a bit more information about the linked demo instead? We are usually in favor of adding a brief explanation for each document or demo linked in the docs.

Maybe,
```suggestion
        For more details on using this operator in applications, see the
        :doc:`SU(N) gate demo <demos/tutorial_here_comes_the_sun>`
        for an in-depth demonstration.
```

Reply:
Sure!